### PR TITLE
Reuse test code

### PR DESCRIPTION
**Pull request by leshikus**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -44,8 +46,49 @@ jobs:
...
+
+      - name: Run smoke tests
+        if: inputs.test == 'smoke'
         run: |
           sudo sh -c 'echo 2 >/proc/sys/vm/overcommit_memory'
-          $CONDA/bin/conda run -n omnisci-dev bash scripts/conda/test.sh
+          conda run -n ${{ env.CONDA_ENV }} bash scripts/conda/test.sh
+
+      - name: Download perf results
+        uses: dawidd6/action-download-artifact@v2
```
Comment (Garra1980) [inline .github/workflows/test.yml:67]: Could you please a comment here explaining why we use this third-party resource

Comment (leshikus) [reply .github/workflows/test.yml:68]: Added

Comment (leshikus) [reply .github/workflows/test.yml:68]: Added the comment

**Review by Garra1980 — APPROVED** [review]
Ok, thanks!

**Pull request merged**